utils/build.ps1

```diff
@@ -1092,24 +1157,16 @@ function Test-SCCacheAtLeast([int]$Major, [int]$Minor, [int]$Patch = 0) {
   return [int]$Matches.3 -ge $Patch
 }
 
-enum Platform {
-  Windows
-  Android
-}
-
-function Get-PlatformRoot([Platform] $Platform) {
-  return ([IO.Path]::Combine((Get-InstallDir $HostArch), "Platforms", "${Platform}.platform"))
+function Get-PlatformRoot([OS] $OS) {
+  return ([IO.Path]::Combine((Get-InstallDir $HostPlatform), "Platforms", "$($OS.ToString()).platform"))
 }
 
-function Get-SwiftSDK {
-  [CmdletBinding(PositionalBinding = $false)]
-  param
-  (
-    [Parameter(Position = 0, Mandatory = $true)]
-    [Platform] $Platform,
-    [string] $Identifier = $Platform.ToString()
-  )
-  return ([IO.Path]::Combine((Get-PlatformRoot $Platform), "Developer", "SDKs", "${Identifier}.sdk"))
+function Get-SwiftSDK([OS] $OS, [switch] $Experimental) {
+  $PlatformStr = $OS.ToString()
+  if ($Experimental) {
+    $PlatformStr += "Experimental"
+  }
```

Comment:
The `Identifier` mechanism is better - especially if we start migrating towards an OS bound thin-sdk approach (i.e. `Windows10.0.17763.0.sdk`, `Windows10.0.26100.0.sdk`, etc).